### PR TITLE
Extend support for Node 22

### PR DESCRIPTION
**Pull request by ellm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "author": "Alley Interactive",
   "license": "GPL-2.0-or-later",
   "engines": {
-    "node": ">=16.0.0 <21.0.0",
+    "node": ">=16.0.0 <22.0.0",
```
Comment (srtfisher) [inline package.json:8]: we want to support 22

```suggestion
    "node": ">=16.0.0 <23.0.0",
```

File: packages/block-editor-tools/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "README.md"
   ],
   "engines": {
-    "node": ">=16.0.0 <21.0.0",
+    "node": ">=16.0.0 <22.0.0",
```
Comment (srtfisher) [inline packages/block-editor-tools/package.json:13]: same

```suggestion
    "node": ">=16.0.0 <23.0.0",
```

File: plugin/package.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "private": true,
   "main": "plugin.php",
   "engines": {
-    "node": ">=16.0.0 <21.0.0",
+    "node": ">=16.0.0 <22.0.0",
```
Comment (srtfisher) [inline plugin/package.json:8]:
```suggestion
    "node": ">=16.0.0 <23.0.0",
```

**Review by srtfisher — CHANGES_REQUESTED** [review]

**Review by srtfisher — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ellm) [reply package.json]: Ah yup - thanks for flagging that. Running a bit on empty today 🫤

**Pull request merged**